Tasks/UseGoV1/usego.ts

```diff
@@ -0,0 +1,23 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as proxyutil from './proxyutil';
+import * as installer from './installer';
+
+async function run() {
+    try {
+        let version = tl.getInput('version', false);
+        if (version) {
+            await installer.getGo(version.trim());
+        }
+
+        // Note - proxy will not work with go get since that is done through source control providers.
```

Comment:
I want to call out this behavior. I think this is reasonable since we don't know which source control provider they're going to use, but there are other options here (aka best effort for common source control providers)